src/configure_config.cmake

```diff
@@ -100,3 +100,6 @@ endif()
 if (TARGET ch_contrib::jemalloc)
     set(USE_JEMALLOC 1)
 endif()
+if (TARGET ch_contrib::mimalloc)
+    set(USE_MIMALLOC 1)
```

Comment:
You forget to add the following to https://github.com/ClickHouse/ClickHouse/blob/master/src/Common/config.h.in:

```
#cmakedefine01 USE_MIMALLOC
```